frontend/apps/vara-man/src/components/layout/header/header.tsx

```diff
@@ -1,33 +1,49 @@
-import { useAccount } from '@gear-js/react-hooks';
-import { Link } from 'react-router-dom';
-import { useGame } from '@/app/context/ctx-game';
-import { HeaderAdmin } from '@/components/layout/header/header-admin';
-import { Icons } from '@/components/ui/icons';
-import { useApp } from '@/app/context/ctx-app';
+import { Link } from 'react-router-dom'
+import { useGame } from '@/app/context/ctx-game'
+import { HeaderAdmin } from '@/components/layout/header/header-admin'
+import { Icons } from '@/components/ui/icons'
 
-import { WalletNew } from '@dapps-frontend/ui'
+import { Header as CommonHeader, MenuHandler } from '@dapps-frontend/ui'
+import {
+	EzGaslessTransactions,
+	EzSignlessTransactions,
+	useEzTransactions,
+} from '@dapps-frontend/ez-transactions'
+import { useAccount, useAlert } from '@gear-js/react-hooks'
 
-export const Header = () => {
-  const { isSettled } = useApp();
-  const { isAdmin } = useGame();
-  const { account } = useAccount();
-
-  return (
-    <header className="container flex justify-between items-center py-7.5">
-      <Link to="/" className="inline-flex text-white transition-colors hover:text-opacity-70">
-        <Icons.logo className="h-15" />
-      </Link>
+import styles from './Header.module.scss'
+import { SIGNLESS_ALLOWED_ACTIONS } from '@/app/consts'
 
-      {account && isSettled && (
-        <div className="flex space-x-4 ml-auto">
-          {isAdmin && <HeaderAdmin />}
-        </div>
-      )}
-
-      <div className="ml-4 text-black">
-        <WalletNew />
-      </div>
+export const Header = () => {
+	const { isAdmin } = useGame()
+	const { account } = useAccount()
```

Comment:
Is it need?
'account' is assigned a value but never used
'gasless' is assigned a value but never used
'signless' is assigned a value but never used
'alert' is assigned a value but never used